src/App.js

```diff
@@ -14,11 +16,110 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    goods: goodsFromServer,
+    startedList: false,
+    reversed: false,
+    sortBy: '',
+  };
+
+  isStarted = () => {
+    this.setState({ startedList: true });
+  };
+
+  resetList = () => {
+    this.setState({
+      sortBy: '',
+      reversed: false,
+    });
+  };
```

Comment:
Let's make this function do what it actually should do.
If it's called 'reset' so it should return the original list (not sorting 'by nothing', as you have written).
You can just assign the original 'goodsFromServer' to your state here

Reply:
done)